docs/rest_api/getting_started.rst

```diff
@@ -0,0 +1,99 @@
+Getting started
+###############
+
+Mautic provides several ``REST API`` endpoints that you can use. In the navigation menu on the left, you can find the available endpoints.
+
+.. warning:: 
+
+    By default, Mautic's ``REST API`` isn't enabled. The Mautic administrator can enable the API in the Mautic UI under ``Configuration -> API Settings``, or by setting ``'api_enabled' => 1`` in ``app/config/local.php`` directly.
+
+.. note:: 
+
+    Mautic has an API library available for PHP. You can :xref:`find it on GitHub<Mautic API Library>`.
+
+To get started easily, you can use Mautic's Postman collection:
+
+.. image:: https://run.pstmn.io/button.svg
+   :target: https://app.getpostman.com/run-collection/19345380-9b7bbddc-8a4d-437a-8fc2-42b0b9823883?action=collection%2Ffork&source=rip_markdown&collection-url=entityId%3D19345380-9b7bbddc-8a4d-437a-8fc2-42b0b9823883%26entityType%3Dcollection%26workspaceId%3D2c328b62-2531-4e35-a6bc-0f2995ce2df3
```

Comment:
I thought this might be a nice addition. It adds the "Run in Postman" button to the docs:

<img width="707" alt="image" src="https://github.com/mautic/developer-documentation-new/assets/17739158/8242aad8-f6eb-4f26-be02-782b102faacf">

It's in a public Postman collection, so we can add admins etc. to collaborate on it.